pkgs/top-level/all-packages.nix

```diff
@@ -3349,6 +3349,8 @@ with pkgs;
 
   hylafaxplus = callPackage ../servers/hylafaxplus { };
 
+  hyperscan = callPackage ../development/libraries/hyperscan { };
```

Comment:
This would be better located between `hyena` and `icu58` in the `### DEVELOPMENT / LIBRARIES` section.
(nitpick)